submissions/Akshata-4-2021-other/Akshata-4-2021-other.md

```diff
@@ -1 +1,49 @@
+
+### Overview
+
+The greatest treasure mankind can save for its future generations is knowledge. Knowledge in any form should be secured and never be lost. This project focuses on scientific knowledge of any form including research papers, preprints, writeups and thesis which could be open source. It will be built on 'Arweave' which assures that the papers uploaded on the platform will never be lost (would stay for thousands of years without any additional costs) and it is decentralized. 
+
+One other major goal of this project is that you can donate crypto tokens to the research paper, preprints, articles, thesis etc. which you find helpful. It is also possible to keep your papers public or private depending upon whether you want to safegaurd your work for yourself or if you like it to be open in public domain. The future goal would be to create a token called 'scholar' wich could be donated to important research projects asking for funds. As the platform will be built on Arweave, the necessary information and guidance about the token is already available.
+
+### Implementation Plan
+
+The first goal of this project is to create a platform which makes the uploads, donations, making papers public or private on web 3 possible. The first task of creating the frontend is already in the process. This project is using React.js and Arweave.js to make the project on web 3 possible. 
+
+If time permits, the next task would be to create 'scholar' tokens within a given timeframe. I am fully dedicated to this project and would be working full time solely for this project during the 12 weeks. 
+
+### Minimum Deliverables
+
+The minimum outcome of this research is to make the platform ready with all the funtionalities mentioned above. This would be possible as the work in already in progress. 
+
+### Aim/Deliverable 1
+
+I would focus on completing the platform within the first 10 weeks. ANd keep the rest of the time for uploads and checking for bugs. All the progress will be continuously reported to the mentor.
+
+### Aim/Deliverable 2
+
+The next aim of the project is to create tokens to reward publications and research projects that needs fundings. 
+
+### Timeline
+
+As the project is already in making, all the updates and progress about the project will be reported to the mentor twice a week. 
+
+### Plans to Continue with Project
+
+This project is the primary thing I am working on and would continue working on the mission of DeSci in future.
+
+### Candidate Details
+
+### Motivation
+
+I strongly believe that Science community should shift to web3 and blockchain. It would reduce the barrier and open doors for greater opportunities which would never be possible in web2. 
+
+Background
+
+I studied Physics major and Mathematics minor while persuing my Masters in Science. After that I worked as  a full stack developer and a data Scientist for year and a half. I am currently a kernel fellow. 
```

Comment:
please consider leaving a social media link, porfolio, writing sample or other piece of information to verify your background, it's not a requirement but it helps establish trust in the community during review